source/administration/changelog.md

```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.30
+
+### Compatibility
+  - PostgreSQL ended long-term support for [version 9.4 in February 2020](https://www.postgresql.org/support/versioning). Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within [the PostgreSQL documentation](https://www.postgresql.org/support/versioning/).
+
+### Highlights
+
+#### Incident Management provided out-of-the-box (E20)
+ - Pre-packaged and pre-installed the Incident Management as well as Channel Export plugins for enterprise-ready builds.
+
+#### Configure New Admin Roles Permissions in the System Console (E20, Beta)
+ - Mattermost [recently released three new pre-built granular administrator roles](https://mattermost.com/blog/mattermost-release-v5-28/) to enable you to selectively delegate administrative tasks to other members of your organization. The three new roles are System Manager, User Manager, and Read-only Admin. Now you can configure specific permissions for these roles directly from the System Console.
+
+### Improvements
+
+#### User Interface
+ - @-autocomplete results are now prioritized based on recency and thread activity.
+ - File attachments below the size of 10 (KB, MB, GB, TB, etc.) now allow showing fractions.
+ - The formatting of the channel header change message was improved.
+ - Team invite workflow now shows BOT tags when the search returns Bot users.
+ - Added the ability to zoom in and out of PDF files.
+ - Added support for 16x16 base64 encoded mini images to use with progressive rendering.
+
+#### Notifications
+ - Channel-wide mentions are now automatically disabled when a user mutes a channel.
+
+#### Command Line Interface (CLI)
+ - Added new local API endpoints for getting, updating and deleting incoming and outgoing webhooks.
+ - Added ``mmctl system version`` endpoint to print the remote server version.
+ - Improved the ``mmctl system status`` command output to include all the reported values.
+
+#### Integrations
+ - Updated icon_emoji field in incoming webhooks to allow emojis to be specified with surrounding colons.
+ - Dynamic auto-completion is now supported for built-in slash commands.
+ - Added plugin hooks for ``ReationHasBeenAdded`` and ``ReactionHasBeenRemoved``.
```

Comment:
```suggestion
 - Added plugin hooks for ``ReactionHasBeenAdded`` and ``ReactionHasBeenRemoved``.
```

(This was a typo in the MR title, see https://github.com/mattermost/mattermost-server/pull/15765/files for the source code)